grafana/dashboards/cmode/harvest_dashboard_network_detail.json

```diff
@@ -541,1984 +541,2570 @@
+                "lastNotNull",
+                "max"
+              ],
+              "displayMode": "table",
+              "placement": "bottom"
+            },
+            "tooltip": {
+              "mode": "single"
             }
-          }
-        },
-        {
-          "id": "merge",
-          "options": {}
+          },
+          "pluginVersion": "8.1.8",
+          "targets": [
+            {
+              "expr": "nic_util_percent{datacenter=\"$Datacenter\",cluster=\"$Cluster\",node=~\"$Node\",nic=~\"$Eth\"}",
+              "interval": "",
+              "legendFormat": "{{node}} {{nic}}",
+              "refId": "A"
+            }
+          ],
+          "timeFrom": null,
+          "timeShift": null,
+          "title": "Port Utilization %",
+          "type": "timeseries"
         },
         {
-          "id": "calculateField",
+          "datasource": "${DS_PROMETHEUS}",
+          "description": "Summarized on Cluster or Node level. Select Eth to show for a specific Ethernet Port.",
+          "fieldConfig": {
+            "defaults": {
+              "color": {
+                "mode": "palette-classic"
+              },
+              "custom": {
+                "axisLabel": "",
+                "axisPlacement": "auto",
+                "barAlignment": 0,
+                "drawStyle": "line",
+                "fillOpacity": 10,
+                "gradientMode": "none",
+                "hideFrom": {
+                  "legend": false,
+                  "tooltip": false,
+                  "viz": false
+                },
+                "lineInterpolation": "linear",
+                "lineWidth": 1,
+                "pointSize": 5,
+                "scaleDistribution": {
+                  "type": "linear"
+                },
+                "showPoints": "auto",
+                "spanNulls": true,
+                "stacking": {
+                  "group": "A",
+                  "mode": "none"
+                },
+                "thresholdsStyle": {
+                  "mode": "off"
+                }
+              },
+              "links": [],
+              "mappings": [],
+              "min": 0,
+              "thresholds": {
+                "mode": "absolute",
+                "steps": [
+                  {
+                    "color": "green",
+                    "value": null
+                  },
+                  {
+                    "color": "red",
+                    "value": 80
+                  }
+                ]
+              },
+              "unit": "none"
+            },
+            "overrides": []
+          },
+          "gridPos": {
+            "h": 8,
+            "w": 8,
+            "x": 8,
+            "y": 32
+          },
+          "id": 29,
           "options": {
-            "alias": "linkSpeed",
-            "binary": {
-              "left": "speed",
-              "operator": "*",
-              "reducer": "sum",
-              "right": "8"
-            },
```

Comment:
This is because we have converted Mbps to Bps(bytes per second) in plugin. [This commit](https://github.com/NetApp/harvest/commit/82328c2831bba1f0743e8977083de099ba09df56)  has reference of comment and dashboard changes. So, we are doing reverse in UI.

One of the ask in [Issue](https://github.com/NetApp/harvest/issues/673)
- Speed is shown in Gbyte instead of Gbit
And we have handled in 2 places cmode+7mode network dashboards.